.travis.yml

```diff
@@ -3,7 +3,6 @@ language: node_js
 node_js:
   - "0.10.40"
 install:
-  - "npm install -g eslint"
-  - "npm install -g eslint-plugin-meteor"
+  - "npm install"
 script:
-  - "eslint ./"
+  - "npm test"
```

Comment:
Hum, I am pretty sure when I tested yesterday `npm test; echo $?` always gave me `0` even if there was some errors, but as I tested it again today I can’t reproduce it, so I suppose I was doing something wrong... Anyway I prefer the output with the `--silent` option because I don’t see the point of having 20 lines of `npm ERR!` to explain that is probably a problem with Wekan and not with NPM.

About Meteor integration, IIRC one argument if favor of it is that it print lint warning in development server started by `meteor run`, so that’s would definitely improve the contributor experience that won’t have to write a lot of LOC before realizing they using different code styling conventions.

Also ESLint recently introduced a `--fix` option to automatically fix trivial formating issues. How could we use that tool if the command line `eslint` refer to the global version that misses some plugins instead of using the local one?